exonum/src/api/backends/actix.rs

```diff
@@ -108,6 +106,14 @@ impl ServiceApiBackend for ApiBuilder {
     }
 }
 
+// FIXME: workaround for a regression in actix-web 0.7.3 (ECR-2149)
```

Comment:
We don't have such conventions, but personally I'm using "FIXME" to mark things that should be fixed _before_ creating pull request. I propose to change it into "TODO".